### PR TITLE
added min/max in LM/R2 sigmas

### PR DESCRIPTION
**Pull request by rjbaraldi**

### CODE REVIEW

**Review by dpo — CHANGES_REQUESTED** [review]

File: src/LM_alg.jl [hunk 1]
```diff
@@ -54,6 +54,8 @@ function LM(
   verbose = options.verbose
   maxIter = options.maxIter
   maxTime = options.maxTime
+  σmax = options.σmax
```
[the remark this answers is not in the document]
Comment (rjbaraldi) [reply src/LM_alg.jl:57]: Oh this isn't common practice too? I'll get rid of it.

**Comment by rjbaraldi** [comment]
Looks like the failure was with Arpack. Do you want to push this through?

File: src/LM_alg.jl [hunk 2]
```diff
@@ -54,6 +54,7 @@ function LM(
   verbose = options.verbose
   maxIter = options.maxIter
   maxTime = options.maxTime
+  σmin = options.σmin
```
Comment (dpo) [inline src/LM_alg.jl:57]: We need to be sure that σ0≥ σmin.

File: src/R2_alg.jl [hunk 3]
```diff
@@ -78,6 +78,7 @@ function R2(
   verbose = options.verbose
   maxIter = options.maxIter
   maxTime = options.maxTime
+  σmin = options.σmin
```
Comment (dpo) [inline src/R2_alg.jl:81]: here too

**Review by dpo — CHANGES_REQUESTED** [review]

File: src/LM_alg.jl (hunk 2 above)
Comment (rjbaraldi) [reply src/LM_alg.jl:56]: I think this is actually a bit later where sigmak is defined but yes good catch.

File: src/R2_alg.jl (hunk 3 above)
Comment (rjbaraldi) [reply src/R2_alg.jl:81]: ditto

**Pull request closed without merging**